wiki/Tutorials/Loadbalancer/index.md

```diff
@@ -0,0 +1,161 @@
+```
+
+Replace the </internal_ip_address_of_web_service>/ with the actual internal IP address of your web-service (e.g. 10.10.0.232), the port should be the one you used for your web-app. When all ist setup, restart the nginx service:
+```console
+sudo systemctl restart nginx
+```
+
+## Security groups port
+
+Before you can reach the web-service from the reverse-proxy the security group of the web-service must be configured with the ip address of the reverse-proxy and the defined port. 
+To create a new rule click on 'Network' and select 'Security Groups' here click on 'Manage Rules' for the security group associated to your web-service. 
+
+![select security group](images/16_select_security_group.png)
+
+Click on 'Add Rule' in the next window and fill in the information for the new rule. Select 'Custom TCP Rule' and enter port 8080 (or the port you defined for the web-service). In the section 'CIDR' fill in the IP address of the reverse proxy with the subnet mask length as shown in the image (e.g. 10.0.2.71/24). Now click on 'Add' to add the rule to the security group. 
+
+
+## DMZ network
+
+To use the reverse-proxy with a load balancer you first need an internal demilitarized zone (dmz) network. This can be created the way you did before. So select 'Network' in the dashboard and then 'Networks' again. In the next window click on 'Create Network' to open the network creation panel.
+
+Give the network the name 'dmz-int' and name the subnet 'dmz-int-subnet'. Now use a private IP address range that is not already in use (e.g. 10.0.10.0/24) and leave everything else free. Click on 'Next' and 'Create' in the last window. When the network is created, you need to create another router which connects the dmz-int network you just created to the dmz-ext network which is created automatically. 
+Click on 'Routers' in the 'Network' section and select 'Create Router'. 
+
+![routers section](images/21_create_dmz_router.png)
+
+Give the route a recognizable name (e.g. dmz-router) and select the network 'dmz-ext' as external network. Now go to 'Routers' in the 'Network' section and go to the 'Interfaces' tab. Here click on 'Add interface' to add another interface for the dmz-int network. 
+
+![router interface](images/23_create_dmz_router_interface.png)
+
+Select the subnet 'dmz-int' and fill in the last possible ip address in this range (e.g.10.0.10.254). Click on 'Submit' to create the interface.
+
+
+## Load-Balancer
+
+Now you can create a load balancer. Click on 'Load Balancer' in the 'Network' section and click on 'Create Load Balancer' to open the creation panel. 
+
+![create load balancer](images/25_create_loadbalancer.png)
+
+Give the load balancer a recognizable name (e.g. web-service-lb) and select the subnet 'dmz-int' you created for the dmz. In the next window you will create the first listener for the load balancer. In this case it is the listener for port 80 or the protocol http. Name it 'http' select 'TCP' as protocol and port 80. 
```

Comment:
It seems a bit missleading to state the listener for port 80 is for protocoll http. So please make it very, very clear that users have to choose the protocoll tcp but advise the user to name their listener "http"